cmd/minikube/cmd/start.go

```diff
@@ -113,6 +113,11 @@ func runStart(cmd *cobra.Command, args []string) {
 	libMachineClient := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 	defer libMachineClient.Close()
 
+	if _, err := shellCfgUnset(libMachineClient); err != nil {
```

Comment:
-1, this does not do anything. All you get back is a reference to ShellConfig. There is nothing happening here. 

So either we really unset the env variables via os.Unsetenv or we don't do anything.